cmd/osm-controller-bootstrap/osm-controller-bootstrap.go

```diff
@@ -33,11 +36,18 @@ import (
 	"github.com/openservicemesh/osm/pkg/version"
 )
 
+const (
+	meshConfigName          = "osm-mesh-config"
+	presetMeshConfigName    = "preset-mesh-config"
+	presetMeshConfigJSONKey = "preset-mesh-config.json"
+)
+
 var (
 	verbosity          string
 	osmNamespace       string
 	caBundleSecretName string
 	osmMeshConfigName  string
+	kubeConfigFilePath string
```

Comment:
This is not required since we use in-cluster config. Let's remove it and other occurrences here.